### ISSUE TITLE
Python3 Error

### ISSUE THREAD

**Issue by BuSHari** [skarppi/icloud_photo_station#3]
Hi, i'm trying to use python3 to run the script to check if it will run better, i think that you have an error in the file 'filesystem.py'

here is the error i'm getting:
```
Traceback (most recent call last):
  File "/volume1/@appstore/icloud_photo_station/app/download_photos.py", line 11, in <module>
    from filesystem import FileSystemStorage
  File "/volume1/@appstore/icloud_photo_station/app/filesystem.py", line 55
    print "Deleting %s!" % self.path
                       ^
SyntaxError: Missing parentheses in call to 'print'

```

**Comment by skarppi** [comment]
Python 3 support is untested but shouldn't be too hard to get fixed. 

I'm also using mostly PhotoStation storage and FileSystem storage might be not be fully working in all use cases. I'll try to get it up to date also in the next release.

**Comment by BuSHari** [comment]
thank you, i think you only need to fix 
`print "Deleting %s!"`
to
`print ("Deleting %s!")`

**Comment by skarppi** [comment]
This bug is now fixed in the latest 0.1.5 release.
